bin/ncar_run.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
-# ./build/release/bin/samurai does not work somehow
-EXE="./build/release/bin/samurai"
+# This script should be executed under /path_to_samurai_root/run directory
```

Comment:
@sjsprecious we need to have the ability to run this script on the CPU using OpenMP threads.  Currently, this script only runs a single GPU or CPU.  If on the CPU we should set the number of OpenMP threads in this file.

Reply:
My bad! I thought we did not want to deal with the OpenMP threading so I just deleted those settings. Add them back now.